sharding-integration-test/sharding-jdbc-test/src/test/java/org/apache/shardingsphere/dbtest/engine/util/IntegrateTestParameters.java

```diff
@@ -67,7 +68,8 @@
             SQLCaseType caseType = (SQLCaseType) each[2];
             Class<?> sqlStatementClass = SQLParseEngineFactory.getSQLParseEngine(
                     DatabaseTypes.getTrunkDatabaseType(databaseType).getName()).parse(sqlCasesLoader.getSQL(sqlCaseId, SQLCaseType.Placeholder, Collections.emptyList()), false).getClass();
-            if (!sqlType.getSqlStatementClass().isAssignableFrom(sqlStatementClass)) {
+            if (!sqlType.getSqlStatementClass().isAssignableFrom(sqlStatementClass)
```

Comment:
The code `SQLType.DML == sqlType && SelectStatement.class == sqlStatementClass` is difficult to understand.
We need consider about it or add some comment to let develop know what happen